include/swift/Basic/Sanitizers.h

```diff
@@ -21,6 +21,8 @@ enum class SanitizerKind : unsigned {
   Thread,
 };
 
+enum class SanitizerCoverageKind : unsigned { None = 0, Function, BB, Edge };
```

Comment:
Nitpick: can you split this onto multiple lines? Putting it all on one line doesn't really buy us anything.